submissions/UA-girl/task_js_dom/js/script.js

```diff
@@ -0,0 +1,186 @@
+        link: 'https://en.wikipedia.org/wiki/Maine_Coon',
+        image: {
+            url: './img/maine_coon.jpg',
+            alt: 'Maine Coon cat image'
+        }
+    },
+
+    {
+        id: 3,
+        title: 'Sphynx cat',
+        text: 'The Sphynx cat is a breed of cat known for its lack of coat (fur). Hairlessness in cats is a ' +
+        'naturally occurring genetic mutation; however, the Sphynx cat, as a breed, was developed through selective breeding, ' +
+        'starting in the 1960s. The skin should have the texture of chamois leather, as it has fine hairs, ' +
+        'or the cat may be completely hairless. Whiskers may be present, either whole or broken, or may be totally absent. ' +
+        'The cats have a narrow, long head and webbed feet. Their skin is the color that their fur would be, and all ' +
+        "the usual cat markings (solid, point, van, tabby, tortie, etc.) may be found on the Sphynx cat's skin. " +
+        "Because they have no fur they lose more body heat than coated cats which makes them warm to the touch and prone to finding warm places...",
+        link: 'https://en.wikipedia.org/wiki/Sphynx_cat',
+        image: {
+            url: './img/canadian_sphinx.jpg',
+            alt: 'Canadian Sphynx cat image'
+        }
+    },
+
+    {
+        id: 4,
+        title: 'Turkish Angora',
+        text: "The Turkish Angora (Turkish: Ankara kedisi, 'Ankara cat') is a breed of domestic cat. Turkish Angoras " +
+        "are one of the ancient, natural breeds of cat, having originated in central Turkey, in the Ankara region. " +
+        "The breed has been documented as early as the 17th century and is believed to be the origin of the mutations " +
+        "for both the color white[citation needed] and long hair[citation needed]. " +
+        "The breed is also sometimes referred to as simply the Angora or Ankara cat...",
+        link: 'https://en.wikipedia.org/wiki/Turkish_Angora',
+        image: {
+            url: './img/angorra.jpg',
+            alt: 'Angorra cat image'
+        }
+    },
+];
+
+const menu = document.querySelector('.catalog-list');
+const menuItems = document.querySelectorAll('.catalog-link');
+const card = document.querySelector('.card');
+
+// menuItems.forEach(item => item.addEventListener('click', toggleCatalog));
+menu.addEventListener('click', toggleCatalog);
+menu.addEventListener('keyup', function (event) {
+    if (event.keyCode === 32) {
+        toggleCatalog(event);
+    }
+});
+
+// Animate appearance of the card
+function animate(item) {
+    let opacity = +item.style.opacity;
+    if (opacity < 1) {
+        opacity += 0.05;
+        item.style.opacity = opacity.toString();
+        setTimeout(animate, 50, item);
+    }
+}
+
+
+// Fill card
+function createPattern(item, node) {
+    // Update image
+    const img = node.querySelector(`.${IMG_CLASS_NAME}`);
+    img.setAttribute('alt', item.image.alt);
```

Comment:
You can destructure the `item` object. More info [here](https://developer.mozilla.org/uk/docs/Web/JavaScript/Reference/Operators/%D0%94%D0%B5%D1%81%D1%82%D1%80%D1%83%D0%BA%D1%82%D1%83%D1%80%D0%B8%D0%B7%D0%B0%D1%86%D1%96%D1%8F)